src/cmd/itn_uptime_analyzer/main_itn_uptime_analyzer.go

```diff
@@ -1,16 +1,17 @@
 package main
 
 import (
-    dg "block_producers_uptime/delegation_backend"
-    itn "block_producers_uptime/itn_uptime_analyzer"
-    "context"
+	dg "block_producers_uptime/delegation_backend"
+	itn "block_producers_uptime/itn_uptime_analyzer"
+	"context"
+	"fmt"
 	"os"
-    "fmt"
+	"strings"
 
-    "github.com/aws/aws-sdk-go-v2/config"
-    "github.com/aws/aws-sdk-go-v2/service/s3"
-    "github.com/aws/aws-sdk-go/aws"
-    logging "github.com/ipfs/go-log/v2"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go/aws"
+	logging "github.com/ipfs/go-log/v2"
```

Comment:
4 spaces per indentation? I thought python era has ended already

Reply:
Oh yeah, I haven't participated in a proper indentation war for ages now. Let me fetch my armor from the basement. Hold on!

More seriously, I think it might be my fault. I didn't take time to configure my Emacs for Go properly, I realised at some point that it indents with tabs rather than spaces. Because the project does not have a proper linter, I decided to quickly fix that using `sed` to replace any tab with 4 spaces. Feel free to change any 2 spaces into one to fix that. Sorry for the mess!

Reply:
I use the [vim-go](https://github.com/fatih/vim-go) plugin for vim to do my formatting and I have `set tabstop=2 and set shiftwidth=2` set. I have disabled the formatter for these changes but it STILL formatted the imports. Otherwise it would have done things to the whole code.